java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -66,6 +67,7 @@ class BindingContext(
     private val coroutineContext =
         parentCoroutineContext + job + CoroutineName("BindingContext-$id")
 
+    @ExperimentalCoroutinesApi
```

Comment:
Any reason not to just also mark the entire class? (same question below)

Reply:
Done here and for the partition map below. It propagated the annotation to a few add'l places but soon it won't be experimental any more, one hopes, anyway.